core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -637,6 +638,14 @@ protected static void assertTermQuery(Query query, String field, String value) {
         assertThat(termQuery.getTerm().text().toLowerCase(Locale.ROOT), equalTo(value.toLowerCase(Locale.ROOT)));
     }
 
+    protected static void assertPrefixQuery(Query query, String field, String value) {
+        assertThat(query, instanceOf(PrefixQuery.class));
+        PrefixQuery prefixQuery = (PrefixQuery) query;
+        assertThat(prefixQuery.getPrefix().field(), equalTo(field));
+        assertThat(prefixQuery.getPrefix().text().toLowerCase(Locale.ROOT), equalTo(value.toLowerCase(Locale.ROOT)));
```

Comment:
Simple copy/paste ;) This is not needed anyway and I guess it was there because this function is supposed to check queries generated from an analyzer. I'll remove it.